packages/chopsticks/src/plugins/follow-chain/index.ts

```diff
@@ -54,33 +61,80 @@ export const cli = (y: Argv) => {
             const header = registry.createType<Header>('Header', data)
             const wasm = await parent.wasm
 
-            const block = new Block(chain, header.number.toNumber(), header.hash.toHex(), parent)
+            const block = new Block(chain, header.number.toNumber(), header.hash.toHex(), parent, {
+              storage: parent.storage,
```

Comment:
you can include header as well so it doesn't have to fetch again
```suggestion
            const block = new Block(chain, header.number.toNumber(), header.hash.toHex(), parent, {
              header,
              storage: parent.storage,
```